backend/src/EntityExtractor.Web/Models/UploadFileRequestModel.cs

```diff
@@ -0,0 +1,4 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace eKultura.EntityExtractor.Web.Models;
+public record UploadFileRequestModel(IFormFile File, string Topic);
```

Comment:
Please add a new line between namespace and class declaration so it doesn't blend together